.github/workflows/trigger-gitlab-pipeline.yml

```diff
@@ -1,25 +1,88 @@
 name: Trigger GitLab CI Pipeline
 
 on:
-    push:
-        branches:
-            - master
+  push:
+    branches:
+      - master
+      - "release/**"
```

Comment:
From review session: This will cover the published branches too which is unintended.

Reply:
`- "release/**"
  - "!release/**-published"`

fix has been added to negate published branches